src/accelerate/state.py

```diff
@@ -106,7 +106,7 @@ class PartialState:
           in use.
         - **local_process_index** (`int`) -- The index of the current process on the current server.
         - **mixed_precision** (`str`) -- Whether or not the current script will use mixed precision, and if so the type
-          of mixed precision being performed.
+          of mixed precision being performed. (One of `"fp16"`, `"bf16"`, or `"fp8"`).
```

Comment:
Can't be no here? If that's the case the sentence before should be rephrased.

Reply:
Rephrasedd to fully mimic the version in the accelerator